### ISSUE TITLE
Release a arm64 version of the image to take advantage of the Pi 4 hardware.

### ISSUE THREAD

**Issue by ghost** [LeoRover/leo_os#10]
Hi, 

I'm wondering if it would be possible to release a 64 bit version of the LeoOS?

I'm trying to run the webrtc_ros package compiled for armhf, this version seems corrupted and it crashes. I'd like to be able to run the arm64 version of this ROS package since I tested it on a Jetson Nano (aarch64 with the arm64 ros pkg) and it's working fine.

**Comment by alex-ssom** [comment]
I'll give it a try soon!

**Comment by alex-ssom** [comment]
I don't know if my internet connection is the issue, but I got quite a few `apt` errors and `rosdep init` didn't work. I'll retry on monday

**Comment by alex-ssom** [comment]
The image is quite unstable. I'm having a `sudo apt update` lock error message if I run the update command too quickly after ssh. ROS master isn't launched (but this might be caused by the fact that I'm testingon a Pi4 outside of the rover (no AP usb adapter)

**Comment by bjsowa** [comment]
Without the External Wifi USB card, ROS Master won't be launched. This is also the case with the previous releases of LeoOS and there is not much I can do about it.

Regarding the lock error on apt update:
There is the `apt-daily` Systemd timer which activates once a day and automatically updates package list. I decided to leave this timer enabled in LeoOS. You can manually disable it by typing:
```
sudo systemctl disable apt-daily.timer
```

**Comment by alex-ssom** [comment]
I just tested it again. I wasn't able to run the `lsusb` command. And I found out that the new Qt display doesn't provide a NetworkManager GUI

**Comment by alex-ssom** [comment]
I suggest that [nm-tray](https://github.com/palinek/nm-tray) be pre-installed.

**Comment by bjsowa** [comment]
I added the usbutils package so the `lsusb` command should be available with the next release.

**Comment by alex-ssom** [comment]
I found an other issue, when running `apt upgrade`:
```bash
Unpacking linux-firmware (1.187.20) over (1.187.19) ...
dpkg: error processing archive /tmp/apt-dpkg-install-zdgoMh/7-linux-firmware_1.187.20_all.deb (--unpack):
 trying to overwrite '/lib/firmware/brcm/brcmfmac43430-sdio.bin', which is also in package linux-firmware-raspi2 4-0ubuntu0~20.04.1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
update-initramfs: Generating /boot/initrd.img-5.4.0-1047-raspi
Using DTB: bcm2711-rpi-4-b.dtb
Installing /lib/firmware/5.4.0-1047-raspi/device-tree/broadcom/bcm2711-rpi-4-b.dtb into /boot/dtbs/5.4.0-1047-raspi/./bcm2711-rpi-4-b.dtb
Taking backup of bcm2711-rpi-4-b.dtb.
Installing new bcm2711-rpi-4-b.dtb.
Installing /lib/firmware/5.4.0-1047-raspi/device-tree/broadcom/bcm2711-rpi-4-b.dtb into /boot/dtbs/5.4.0-1047-raspi/./bcm2711-rpi-4-b.dtb
Taking backup of bcm2711-rpi-4-b.dtb.
Installing new bcm2711-rpi-4-b.dtb.
flash-kernel: deferring update (trigger activated)
update-initramfs: Generating /boot/initrd.img-5.4.0-1044-raspi
Using DTB: bcm2711-rpi-4-b.dtb
Installing /lib/firmware/5.4.0-1044-raspi/device-tree/broadcom/bcm2711-rpi-4-b.dtb into /boot/dtbs/5.4.0-1044-raspi/./bcm2711-rpi-4-b.dtb
Taking backup of bcm2711-rpi-4-b.dtb.
Installing new bcm2711-rpi-4-b.dtb.
Ignoring old or unknown version 5.4.0-1044-raspi (latest is 5.4.0-1047-raspi)
Errors were encountered while processing:
 /tmp/apt-dpkg-install-zdgoMh/7-linux-firmware_1.187.20_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by bjsowa** [comment]
Could you try running `sudo apt install --fix-broken` and then `sudo apt upgrade` again?

**Comment by bjsowa** [comment]
I think I identified the problem with the linux-firmware-raspi2 package. It should be fixed with 0332393

To fix this on your system, just reinstall this package:
```
sudo apt reinstall linux-firmware-raspi2
```
and you should be able to complete the upgrade

**Comment by alex-ssom** [comment]
It didn't fix the issue, but I'll just flash the release candidate image.

**Comment by bjsowa** [comment]
I had to focus on some other things the past couple of months but recently I came back this project and published a [new release candidate](https://github.com/LeoRover/LeoOS/releases/tag/1.0.0rc2). This time we are planning to publish the official release within a week so it is the last call to hunt some bugs.

> Without the External Wifi USB card, ROS Master won't be launched.

This is not the case anymore by the way.
